osu.Game/Screens/Multi/Components/BeatmapTitle.cs

```diff
@@ -53,7 +53,9 @@ private void updateText()
 
             textFlow.Clear();
 
-            if (CurrentBeatmap.Value == null)
+            var beatmap = CurrentItem.Value?.Beatmap;
```

Comment:
This seems to have broken for `DrawableRoom` - it should display above the "Public, Welcoming Players" text.